lib/SimplePie/SimplePie/File.php

```diff
@@ -113,11 +113,11 @@ public function __construct($url, $timeout = 10, $redirects = 5, $headers = null
 				curl_setopt($fp, CURLOPT_REFERER, SimplePie_Misc::url_remove_credentials($url));
 				curl_setopt($fp, CURLOPT_USERAGENT, $useragent);
 				curl_setopt($fp, CURLOPT_HTTPHEADER, $headers2);
-				if (!ini_get('open_basedir') && !ini_get('safe_mode') && version_compare(SimplePie_Misc::get_curl_version(), '7.15.2', '>='))
-				{
-					curl_setopt($fp, CURLOPT_FOLLOWLOCATION, 1);
-					curl_setopt($fp, CURLOPT_MAXREDIRS, $redirects);
-				}
+				//if (!ini_get('open_basedir') && !ini_get('safe_mode') && version_compare(SimplePie_Misc::get_curl_version(), '7.15.2', '>='))
+				//{
+				//	curl_setopt($fp, CURLOPT_FOLLOWLOCATION, 1);
+				//	curl_setopt($fp, CURLOPT_MAXREDIRS, $redirects);
+				//}
```

Comment:
In git leaving commented code is not a good practise. This should probably be removed.

Reply:
See https://github.com/simplepie/simplepie/pull/660 (which I will merge back when accepted)

Reply:
https://github.com/FreshRSS/FreshRSS/pull/3206